### PR TITLE
Installing virtualization components

### PR DESCRIPTION
**Pull request by tbazant**
This new article describes how to install virtualization environment.
Rendered PDF is attached 
[task-install-virtualization-components_color_en.pdf](https://github.com/SUSE/doc-modular/files/7519854/task-install-virtualization-components_color_en.pdf)

!!! @tbazant backport to the original Virtualization guide !!!

### CODE REVIEW

File: xml/task-install-virtualization-components.xml [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+   <para>
+    Related software packages from &productname; software repositories are
+    organized into <emphasis>installation patterns</emphasis>. You can use
+    these patterns to install a specific virtualization components on an
+    already running &productname;. Use <command>zypper</command> to install
+    them:
+   </para>
+<screen>zypper install -t pattern <replaceable>PATTERN_NAME</replaceable></screen>
+   <para>
+    To install &kvm; environment, consider the following patterns:
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:229]: To install the &kvm; environment

File: xml/task-install-virtualization-components.xml [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+     <listitem>
+      <para>
+       Installs &libvirt; tools for managing and monitoring &vmguest;s in &kvm;
+       environment.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+   <para>
+    To install &xen; environment, consider the following patterns:
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:251]: To install the &xen; environment

File: xml/task-install-virtualization-components.xml [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+      <imagedata fileref="virt-system-roles.png" width="75%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="virt-system-roles.png" width="75%"/>
+     </imageobject>
+    </mediaobject>
+   </figure>
+   <para>
+    Here you can select either the <guimenu>KVM Virtualization Host</guimenu>
+    or <guimenu>Xen Virtualization Host</guimenu> roles. The appropriate
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:112]: or the `<guimenu>`Xen

File: xml/task-install-virtualization-components.xml [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+   </para>
+   <tip>
+    <para>
+     The <emphasis>&yast; Virtualization</emphasis> module is included in the
+     <package>yast2-vm</package> package. Verify it is installed on the
+     &vmhost; before installing virtualization components.
+    </para>
+   </tip>
+   <procedure>
+    <title>Installing &kvm; environment</title>
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:140]: Installing the &kvm; environment

File: xml/task-install-virtualization-components.xml [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+    <para>
+     To install &kvm; virtualization environment and related tools, proceed as
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:142]: To install the &kvm; virtualization environment

File: xml/task-install-virtualization-components.xml [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+    </step>
+    <step>
+     <para>
+      After the setup has been finished, you can start creating and configuring
+      &vmguest;s. Rebooting the &vmhost; is not required.
+     </para>
+    </step>
+   </procedure>
+   <procedure>
+    <title>Installing &xen; environment</title>
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:175]: Installing the &xen; environment

File: xml/task-install-virtualization-components.xml [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+    <para>
+     To install &xen; virtualization environment, proceed as follows:
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:177]: To install the &xen; virtualization environment

File: xml/task-install-virtualization-components.xml [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+    </para>
+    <step>
+     <para>
+      Start &yast; and elect<menuchoice> <guimenu>Virtualization</guimenu>
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:181]: Start &yast; and select

File: xml/task-install-virtualization-components.xml [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+     </tip>
+    </step>
+   </procedure>
+  </section>
+  <section xml:id="install-virtualization-components-pattern">
+   <title>Installing specific installation patterns</title>
+   <para>
+    Related software packages from &productname; software repositories are
+    organized into <emphasis>installation patterns</emphasis>. You can use
+    these patterns to install a specific virtualization components on an
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:223]: install specific (no "a")

File: xml/task-install-virtualization-components.xml [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+<screen>zypper install -t pattern <replaceable>PATTERN_NAME</replaceable></screen>
+   <para>
+    To install &kvm; environment, consider the following patterns:
+   </para>
+   <variablelist>
+    <varlistentry>
+     <term><literal>kvm_server</literal></term>
+     <listitem>
+      <para>
+       Installs basic &vmhost; with &kvm; and &qemu; environment.
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:236]: Installs a basic &vmhost; with the &kvm; and &qemu; environments.

File: xml/task-install-virtualization-components.xml [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+   </variablelist>
+   <para>
+    To install &xen; environment, consider the following patterns:
+   </para>
+   <variablelist>
+    <varlistentry>
+     <term><literal>xen_server</literal></term>
+     <listitem>
+      <para>
+       Installs basic &xen; &vmhost;.
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:258]: Installs a basic

File: xml/task-install-virtualization-components.xml [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+    </varlistentry>
+   </variablelist>
+  </section>
+ </section>
+ <section xml:id="summary-install-virtualization-components">
+  <title>Summary</title>
+  <para>
+   After installing virtualization components on the server, you will be able
+   to create, run, and manage virtual machines using preferred virtualization
+   environment.
```
Comment (cjschroder) [inline xml/task-install-virtualization-components.xml:279]: using your preferred

**Review by cjschroder — CHANGES_REQUESTED** [review]
This looks good, other than missing a few 'the' and 'a'.

File: DC-task-install-virtualization-components [hunk 13]
```diff
@@ -0,0 +1,12 @@
+# This file originates from the project https://github.com/openSUSE/doc-kit
+# This file can be edited downstream.
+
+MAIN="task-install-virtualization-components.xml"
+ROOTID="task-example"
```
Comment (cjschroder) [inline DC-task-install-virtualization-components:5]: should be ROOTID="task-install-virtualization-components"

File: xml/task-install-virtualization-components.xml [hunk 14]
```diff
@@ -0,0 +1,353 @@
...
+  <title>Environment</title>
+  <para>
+   This document applies to the following products and product versions:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     &sles;&nbsp;15&nbsp;SP3, 15&nbsp;SP2, 15&nbsp;SP1, 15&nbsp;GA,
+     12&nbsp;SP5, 12&nbsp;SP4, 12&nbsp;SP3
+    </para>
```
[the remark this answers is not in the document]
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:40]: you're right, i'll limit this

File: xml/task-install-virtualization-components.xml [hunk 15]
```diff
@@ -0,0 +1,353 @@
...
+ <section xml:id="environment-install-virtualization-components">
+  <title>Environment</title>
+  <para>
+   This document applies to the following products and product versions:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     &sles;&nbsp;15&nbsp;SP3, 15&nbsp;SP2, 15&nbsp;SP1, 15&nbsp;GA,
+     12&nbsp;SP5, 12&nbsp;SP4, 12&nbsp;SP3
```
Comment (tbazant) [inline xml/task-install-virtualization-components.xml:39]:
```suggestion
     &sles;&nbsp;15&nbsp;SP3
```

File: xml/task-install-virtualization-components.xml (hunk 3 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:112]:
```suggestion
    or the <guimenu>Xen Virtualization Host</guimenu> roles. The appropriate
```

File: xml/task-install-virtualization-components.xml (hunk 4 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:140]:
```suggestion
    <title>Installing the &kvm; environment</title>
```

File: xml/task-install-virtualization-components.xml (hunk 5 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:142]:
```suggestion
     To install the &kvm; virtualization environment and related tools, proceed as
```

File: xml/task-install-virtualization-components.xml (hunk 6 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:175]:
```suggestion
    <title>Installing the &xen; environment</title>
```

File: xml/task-install-virtualization-components.xml (hunk 7 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:177]:
```suggestion
     To install the &xen; virtualization environment, proceed as follows:
```

File: xml/task-install-virtualization-components.xml (hunk 8 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:181]:
```suggestion
      Start &yast; and select<menuchoice> <guimenu>Virtualization</guimenu>
```

File: xml/task-install-virtualization-components.xml (hunk 9 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:223]:
```suggestion
    these patterns to install specific virtualization components on an
```

File: xml/task-install-virtualization-components.xml (hunk 1 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:229]:
```suggestion
    To install the &kvm; environment, consider the following patterns:
```

File: xml/task-install-virtualization-components.xml (hunk 10 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:236]:
```suggestion
       Installs basic &vmhost; with the &kvm; and &qemu; environments.
```

File: xml/task-install-virtualization-components.xml (hunk 2 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:251]:
```suggestion
    To install the &xen; environment, consider the following patterns:
```

File: xml/task-install-virtualization-components.xml (hunk 11 above)
Comment (tbazant) [reply xml/task-install-virtualization-components.xml:258]:
```suggestion
       Installs a basic &xen; &vmhost;.
```

File: xml/task-install-virtualization-components.xml [hunk 16]
```diff
@@ -0,0 +1,353 @@
...
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </section>
+ </section>
+ <section xml:id="summary-install-virtualization-components">
+  <title>Summary</title>
+  <para>
+   After installing virtualization components on the server, you will be able
+   to create, run, and manage virtual machines using preferred virtualization
```
Comment (tbazant) [inline xml/task-install-virtualization-components.xml:278]:
```suggestion
   to create, run, and manage virtual machines using your preferred virtualization
```

**Review by tbazant — COMMENTED** [review]
Agreed on Carla's suggestions, thanks!

**Review by tbazant — COMMENTED** [review]
Suggestions merged

**Pull request merged**